### ISSUE TITLE
Host control commands ignore arguments after the first name

### ISSUE THREAD

**Issue by aanand** [docker/machine#536]
I expected `start` with multiple args to either start all the hosts I specified, or give me an error. Instead, it only started the first host I specified.

```
$ docker-machine ls
NAME           ACTIVE   DRIVER       STATE     URL
host-1                  virtualbox   Stopped
host-2         *        virtualbox   Stopped

$ docker-machine start host-1 host-2
INFO[0000] Waiting for VM to start...

$ echo $?
0

$ docker-machine ls
NAME           ACTIVE   DRIVER       STATE     URL
host-1                  virtualbox   Running
host-2         *        virtualbox   Stopped

```

**Comment by sthulb** [comment]
We only take one name at the moment. For consistency, we should perhaps enforce it with an error message for more than one argument.

**Comment by ehazlett** [comment]
We should also look into the CLI in general.  I think this would make sense to support (much like Docker itself can take multiple container ids).  We are really inconsistent with args across the commands.  I'll add this to the refactor list.

**Comment by nathanleclaire** [comment]
Fixed by #604.  @ehazlett please close when you get a chance.

**Comment by ehazlett** [comment]
Closing as #604 is merged.
